src/Rector/AbstractRector.php

```diff
@@ -118,6 +119,10 @@ abstract class AbstractRector extends NodeVisitorAbstract implements PhpRectorIn
 
     private NodeConnectingTraverser $nodeConnectingTraverser;
 
+    private bool $shouldRemoveCurrentNode = false;
```

Comment:
I think this property is not needed, just use `toBeRemovedNodeHash` as string to verify should be enough.

Reply:
Indeed, thanks